tensorflow/lite/kernels/reverse_test.cc

```diff
@@ -77,6 +77,48 @@ TEST(ReverseOpTest, FloatMultiDimensions) {
                         17, 18, 15, 16, 13, 14, 23, 24, 21, 22, 19, 20}));
 }
 
+TEST(ReverseOpTest, FloatMultiAxes) {
+  ReverseOpModel<float> model({TensorType_FLOAT32, {4, 3, 2}},
+                              {TensorType_INT32, {1}});
+  model.PopulateTensor<float>(model.input(),
+                              {1,  2,  3,  4,  5,  6,  7,  8,  9,  10, 11, 12,
+                               13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24});
+  model.PopulateTensor<int32_t>(model.axis(), {2});
```

Comment:
why call this multiaxes? it seems it's still one axis?

Reply:
@renjie-liu , sorry for the oversight, i have updated the name of the TC, kindly check and approve.